docs/sources/concepts/index.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Learn about concepts basic to tracing.
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/concepts/
+keywords:
+  - Explore Traces
+  - Concepts
+title: Concepts
+menuTitle: Concepts
+weight: 200
+---
+
+# Concepts
+
+Distributed traces provide a way to monitor applications by tracking requests across services.
+Traces record the details of a request to help understand why an issue is or was happening.
+
+Tracing is best used for analyzing the performance of your system, identifying bottlenecks, monitoring latency, and providing a complete picture of how requests are processed.
+
+To use the Explore Traces app, you should understand these concepts:
+
+- [Concepts](#concepts)
+  - [Rate, error, and duration metrics](#rate-error-and-duration-metrics)
+  - [Traces and spans](#traces-and-spans)
+  - [Trace structure](#trace-structure)
```

Comment:
```suggestion
- [Rate, error, and duration metrics](#rate-error-and-duration-metrics)
- [Traces and spans](#traces-and-spans)
- [Trace structure](#trace-structure)
```

I think you can remove the link to Concepts, since it'll probably just scroll the page up?